scripts/release_scripts/update_configs.py

```diff
@@ -144,22 +144,29 @@ def check_updates_to_terms_of_service(
                 f.write(line)
 
 
-def verify_feconf(release_feconf_path, verify_mailgun_api):
+def verify_feconf(release_feconf_path, verify_email_api_keys):
     """Verifies that feconf is updated correctly to include
-    mailgun api key and redishost.
+    mailgun api key, mailchimp api key and redishost.
 
     Args:
-        verify_mailgun_api: bool. Whether to verify mailgun api key.
+        verify_email_api_keys: bool. Whether to verify both mailgun and
```

Comment:
Args don't match the order in function signature

Reply:
Done